Geometry/CSCGeometryBuilder/src/CSCGeometryParsFromDD.cc

```diff
@@ -426,24 +453,31 @@ bool CSCGeometryParsFromDD::build(const cms::DDCompactView* cview,
       }
 
       auto wirespacing = fv.get<double>("WireSpacing");
-      wg.wireSpacing = static_cast<double>(wirespacing);
+      wg.wireSpacing = static_cast<double>(wirespacing / dd4hep::mm);
+      edm::LogVerbatim("CSCGeometryParsFromDD") << "(2) wireSpacing: " << wg.wireSpacing / dd4hep::mm;
```

Comment:
These debug print-out statements are not correct. The first one correctly divides `wirespacing` by 0.1 to put `wg.wireSpacing` into mm.
```
wg.wireSpacing = static_cast<double>(wirespacing / dd4hep::mm);
```
Then 
```
edm::LogVerbatim("CSCGeometryParsFromDD") << "(2) wireSpacing: " << wg.wireSpacing / dd4hep::mm;
```
divides `wg.wireSpacing` by 0.1 again, which is wrong.

Reply:
@cvuosalo : ok, I'll fix them